lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -459,6 +459,13 @@ protected static function userHttpHeader()
         // Provide user and groups to lizmap plugin access control
         $user = $appContext->getUserSession();
         $userGroups = $appContext->aclUserGroupsId();
+        $userPrivateGroup = $appContext->aclUserPrivateGroup();
+        if ($userPrivateGroup) {
```

Comment:
there is no need to do this if statement, if we call the new proposed method `aclPublicGroupsIdByUser()` instead of `aclUserGroupsId()`.